### PR TITLE
Feat(*): SPARK-29316 - cui-button and cui-modal class updates

### PR DESCRIPTION
**Pull request by KHempel**
scss updates for cui-button & cui-modal classes

### CODE REVIEW

**Comment by KHempel** [comment]
Added in tab changes to keep tabs visually unchanged for now

File: scss/settings/modal.scss [hunk 1]
```diff
@@ -27,22 +27,22 @@ $modal-dialog__top: rem-calc(100);
 $modal-dialog__padding: rem-calc(20);
 $modal-dialog__border: 1px solid $black-12;
 $modal-dialog__box-shadow: 0 8px 16px 0 $black-08;
-$modal-header__padding--dialog: 0 0 rem-calc(12) 0;
+$cui-modal__header__padding--dialog: 0 0 rem-calc(12) 0;
```
Comment (pauljeter) [inline scss/settings/modal.scss]: Leave these settings names as is. We use the `cui-` prefix in our classnames, but not in the mixins or variables.

File: scss/tools/mixins/modal.scss [hunk 2]
```diff
@@ -95,14 +95,14 @@
...
-// $font-size - Sets modal header font-size Default: $modal-header__font-size
+// $padding - Sets modal header padding Default: $cui-modal__header__padding
+// $line-height - Sets modal header line height Default: $cui-modal__header__line-height
+// $font-size - Sets modal header font-size Default: $cui-modal__header__font-size
 // $font-family - Sets modal header font-family (i.e. 1px). Default: $brand-font-bold
-@mixin modal-header-size(
-  $padding: $modal-header__padding,
-  $line-height: $modal-header__line-height,
-  $font-size: $modal-header__font-size,
+@mixin cui-modal__header-size(
```
Comment (pauljeter) [inline scss/tools/mixins/modal.scss]: Leave this mixin name as is. We use the `cui-` prefix in our classnames, but not in the mixins or variables.

File: scss/components/modal.scss [hunk 3]
```diff
@@ -27,7 +27,7 @@
       &__header {
         flex-shrink: 0;
 
-        @include modal-header-size;
+        @include cui-modal__header-size;
```
Comment (pauljeter) [inline scss/components/modal.scss]: Leave this mixin name as is. We use the `cui-` prefix in our classnames, but not in the mixins or variables.

File: scss/components/tabs.scss [hunk 4]
```diff
@@ -182,6 +182,24 @@
         padding-bottom: 9.5px;
         border-bottom: 1px solid $gray-5;
         border-radius: 0px;
+        // compatability in Atlas
+        &.nav-link {
```
Comment (pauljeter) [inline scss/components/tabs.scss]: Can we put this in a separate file in the `ng` directory? Trying to keep temporary Atlas changes separate

**Pull request merged**